cmd/syncthing/cli/index.go

```diff
@@ -8,47 +8,31 @@ package cli
 
 import (
 	"github.com/urfave/cli"
-
-	"github.com/syncthing/syncthing/lib/db/backend"
-	"github.com/syncthing/syncthing/lib/locations"
 )
 
 var indexCommand = cli.Command{
 	Name:  "index",
 	Usage: "Show information about the index (database)",
 	Subcommands: []cli.Command{
 		{
-			Name:   "dump",
+			Name:   "indexDump",
 			Usage:  "Print the entire db",
-			Action: expects(0, dump),
+			Action: expects(0, indexDump),
 		},
 		{
-			Name:   "dumpsize",
+			Name:   "indexDumpsize",
```

Comment:
What do we do with multiWord parameters elsewhere? are they camelCased? kebab-cased etc?
Just want to make sure we stay consistent.